.github/workflows/recipe.yaml

```diff
@@ -19,32 +17,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: ['7.4', '8.0', '8.1']
-        sylius: ['~1.8.0', '~1.9.0', '~1.10.0', '~1.11.0', '~1.12.0']
+        php: ['7.4' ,'8.0', '8.1']
+        sylius: ["~1.8.0", "~1.9.0", "~1.10.0", "~1.11.0", "~1.12.0"]
         exclude:
-          - php: 8.0
-            sylius: '~1.8.0'
-          - php: 8.0
-            sylius: '~1.9.0'
           - php: 8.1
-            sylius: '~1.8.0'
+            sylius: "~1.8.0"
           - php: 8.1
-            sylius: '~1.9.0'
+            sylius: "~1.9.0"
+          - php: 8.0
+            sylius: "~1.8.0"
+          - php: 8.0
+            sylius: "~1.9.0"
           - php: 7.4
-            sylius: '~1.11.0'
+            sylius: "~1.11.0"
           - php: 7.4
-            sylius: '~1.12.0'
+            sylius: "~1.12.0"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏why theses changes?

Reply:
Copy paste from Menu :D